turbinia/e2e/setup-gce.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+# This script will
+# - setup a GCE Ubuntu 18.04 LTS base image for Turbinia e2e testing.
+# - execute Terraform to setup the Turbinia environment.
+# Make sure the service account your GCE instance is running under has full API scope
```

Comment:
Similar to the note above, not seeing this below and not sure if it was intended unless the Terraform scripts already does that check?

Reply:
See response above.